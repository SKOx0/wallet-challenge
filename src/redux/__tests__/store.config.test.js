import { browserHistory } from 'react-router-dom';
import configureStore from '../store.config';

describe('Configure Store module', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('Deve ser igual ao snapshot', () => {
    expect(store).toMatchSnapshot();
  });
});
