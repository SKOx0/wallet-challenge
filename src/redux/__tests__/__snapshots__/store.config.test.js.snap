// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Configure Store module Deve ser igual ao snapshot 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
